packages/react/avatar/package.json

```diff
@@ -23,7 +23,8 @@
     "@radix-ui/react-use-layout-effect": "workspace:*"
   },
   "peerDependencies": {
-    "react": "^16.8 || ^17.0 || ^18.0"
+    "react": "^16.8 || ^17.0 || ^18.0",
+    "react-dom": "^16.8 || ^17.0 || ^18.0"
```

Comment:
Will have to re-review these with the change of package so I'm skipping for now.